content/preferences-settings/shortcuts.md

```diff
@@ -5,87 +5,126 @@ weight: 120
 
-The only valid modifiers are the Shift, Ctrl and Alt keys on the keyboard. You can define additional keys (or device buttons) as modifiers by assigning keys/buttons to the "global/modifier" action. However, these will merely function as extra Ctrl, Alt or Shift keys -- you cannot create "new" modifiers.
+If you have an external controller, you can trigger discrete actions by pressing one or more buttons on the controller. Continuous actions can use a combination of buttons and a knob/joystick movement, or just knob/joystick movement.
 
-## extending simple shortcuts with movement
+# anatomy of a shortcut
 
-For certain actions you can choose to _extend_ a simple shortcut using mouse/device movement. For example you might hold Ctrl+X while scrolling with your mouse to change the value of a slider. The following can be used to extend a simple shortcut:
+The simplest shortcut consists of just one key press (e.g., `E`).
 
--   Movement of the mouse scroll wheel
--   Horizontal, vertical or diagonal movement of the mouse cursor
--   Movement of a knob/joystick on an external device
+The same key can be repeated up to three times, so also `E + E` and `E + E + E` are valid, and distinct, shortcuts.
+
+You can further extend a shortcut with up to three clicks of different mouse buttons. So, `E + E + left-click` or `E + right-click + left-click` are also valid shortcuts.
 
-To extend a simple shortcut, you must hold the final key/button of the simple shortcut while performing the extending mouse/device movement.
+Finally, if your fingers are very nimble and you really need to go there, you can repeat the last click up to three times. So, also `E + left-click + middle-click + middle-click` and `E + E + middle-click + left-click + right-click + right-click + right-click` are valid shortcuts.
 
-For external devices you do not need to start with a _simple_ shortcut -- you can directly assign a control knob or joystick to an action -- though this will significantly reduce the flexibility of such devices.
+Hence, the most complex shortcut can consist of 8 key/button presses:
+* Three key presses of the same `key`, followed by
+* Three presses of the three mouse buttons, followed by
+* 2 more repetitions of the last click.
 
-Long button and key presses cannot be extended, as the length of the click/press is timed using the release of the final button/key -- such shortcuts must be terminated with the raising of the final button/key.
+Your shortcut can include one or more modifiers (`Shift`, `Ctrl` and `Alt`). In this case, for a continuous shortcut, both the last key and the modifier must be pressed down when you execute the movement. For example, `Ctrl + E + E + pan` means that while moving the mouse left and/or right both  `E` and `Ctrl` must be pressed down.
 
 ---
 
-**Note:** You may need to switch off the "disable touchpad while typing" setting if you want to use extended shortcuts with a laptop touchpad.
+**Note:**
```

Comment:
This note should probably go in the introductory section, if we include it at all.